types/rethinkdbdash/index.d.ts

```diff
@@ -40,6 +40,7 @@ declare module "rethinkdbdash" {
 
             getAll<T>(value: string, opts?: { index?: string }): Term<T[]>;
             get<T>(id: string): Term<T>;
+            count(query: any): Term<number>
```

Comment:
If we know what should be expected here from the docs this should be typed; not `any`